customTests/azure_nccl_allreduce_ib_loopback.nhc

```diff
@@ -24,13 +24,43 @@ function collect_nccl_allreduce_ib_loopback_data() {
 
 }
 
+function check_dependencies() {
```

Comment:
Is this the same code as the other file? If so, can it be put somewhere shared rather than having to maintain multiple copies?

Reply:
I created a common script that can be sourced on the applicable test scripts.